### PR TITLE
Umer

### PR DESCRIPTION
**Pull request by umershaikh123**
## Changes

- removed mainnet option
- created Dashboad Page
- integrated graphQL fetching

### Dashboard Component
path : packages/arb-token-bridge-ui/src/components/common/DashboardData.tsx
fetches query data and populates the table

### Tab Component
path : packages/arb-token-bridge-ui/src/components/common/Tabs.tsx
Tab is imported from [material ui library](https://mui.com/material-ui/react-tabs/) 
Tab Items are imported from Dashboard Component

### CODE REVIEW

File: packages/arb-token-bridge-ui/src/state/app/utils.ts
Comment (web3cook) [inline packages/arb-token-bridge-ui/src/state/app/utils.ts:1]: remove console.log when pushing to production but for testing its okay

File: packages/arb-token-bridge-ui/src/util/client.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { HttpLink, ApolloClient , InMemoryCache } from "@apollo/client"
+ 
+
+export const getClient = new ApolloClient({
+    cache: new InMemoryCache(),
+    link: new HttpLink({
+      uri: "https://api.studio.thegraph.com/query/58809/nexus/v0.0.1",
```
Comment (web3cook) [inline packages/arb-token-bridge-ui/src/util/client.ts]: put the graph url in env file

**Review by web3cook — CHANGES_REQUESTED** [review]

File: packages/arb-token-bridge-ui/src/util/client.ts (hunk 1 above)
Comment (umershaikh123) [reply packages/arb-token-bridge-ui/src/util/client.ts]: > put the graph url in env file

resolved

File: packages/arb-token-bridge-ui/src/components/App/App.tsx [hunk 2]
```diff
@@ -300,15 +300,23 @@ function AppContent() {
...
+    </div>
       <Header>
         <HeaderAccountPopover />
+
       </Header>
-      <button
-        className=" ml-6 rounded-lg bg-[#ff6600] px-4 py-2 text-sm text-white"
+
+      
+      <Button
```
Comment (umershaikh123) [inline packages/arb-token-bridge-ui/src/components/App/App.tsx:304]: add network button moved to right side and changed its design

File: packages/arb-token-bridge-ui/src/components/Sidebar/SidebarFooter.tsx [hunk 3]
```diff
@@ -14,7 +14,7 @@ export const SidebarFooter = ({ className }: { className?: string }) => {
       )}
     >
       <div className="duration-300  ease-in-out hover:scale-125">
-        <Link href={'https://twitter.com/_Nexus_Network'} target="_blank">
+        <Link href={'https://x.com/NexusNetwork_0x'} target="_blank">
```
Comment (umershaikh123) [inline packages/arb-token-bridge-ui/src/components/Sidebar/SidebarFooter.tsx:17]: updated twitter link

File: packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx [hunk 4]
```diff
@@ -229,7 +229,7 @@ function NetworkContainer({
         <div
           className="absolute left-0 top-0 h-full w-full bg-[-2px_0] bg-no-repeat bg-origin-content p-3 opacity-50"
           style={{
-            backgroundImage,
+            // backgroundImage,
```
Comment (umershaikh123) [inline packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx:232]: removed background logo in bridge background

File: packages/arb-token-bridge-ui/src/util/bridgeUiConfig.ts [hunk 5]
```diff
@@ -46,7 +46,7 @@ export function getBridgeUiConfigForChain(chainId: number): BridgeUiConfig {
         ...ethereumBaseConfig,
         network: {
           ...ethereumBaseConfig.network,
-          name: 'holesky',
+          name: 'Holesky',
```
Comment (umershaikh123) [inline packages/arb-token-bridge-ui/src/util/bridgeUiConfig.ts:49]: corrected holesky name at 
bridge ui
```
From to : network name
```

File: packages/arb-token-bridge-ui/src/util/networks.ts [hunk 6]
```diff
@@ -353,14 +353,14 @@ export function isNetwork(chainId: ChainId) {
```
Comment (umershaikh123) [inline packages/arb-token-bridge-ui/src/util/networks.ts:353]: Removing additional networks displayed at the front end and only displaying nexus and holesky chain

File: packages/arb-token-bridge-ui/src/state/app/utils.ts [hunk 7]
```diff
@@ -104,8 +104,23 @@ export const transformDeposit = (tx: Transaction): MergedTransaction => {
 export const transformWithdrawal = (
   tx: L2ToL1EventResultPlus
 ): MergedTransaction => {
+  console.log('Received transaction:', tx);
   const uniqueIdOrHash = getUniqueIdOrHashFromEvent(tx)
-
+  console.log('Computed uniqueIdOrHash:', uniqueIdOrHash);
+  console.log('tx.sender', tx.sender);
```
Comment (web3cook) [inline packages/arb-token-bridge-ui/src/state/app/utils.ts]: remove consoles

**Review by web3cook — CHANGES_REQUESTED** [review]

File: packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx (hunk 4 above)
Comment (web3cook) [reply packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx:232]: ok

File: packages/arb-token-bridge-ui/src/state/app/utils.ts (hunk 7 above)
Comment (umershaikh123) [reply packages/arb-token-bridge-ui/src/state/app/utils.ts]: resolved

**Review by rishav-eulb — APPROVED** [review]

**Review by web3cook — APPROVED** [review]

**Pull request merged**